src/components/footer/footer.module.scss

```diff
@@ -128,6 +136,8 @@
 	}
 
 	.footerNavigationContainer {
+		position: static;
```

Comment:
better remove unnecessary line if you dont change position here and in other similar places